docs/pages/kubernetes-access/helm/guides/digitalocean.mdx

```diff
@@ -0,0 +1,174 @@
+
+```code
+$ kubectl --namespace teleport-cluster exec deploy/teleport-cluster -- tctl users add tadmin --roles=access,editor --logins=root,ubuntu
+# User "tadmin" has been created but requires a password. Share this URL with the user to complete user setup, link is valid for 1h:
+# https://tele.teleporters.dev:443/web/invite/9b8c8ccb53d0aab612487d279d3d9831
+
+# NOTE: Make sure tele.teleporters.dev:443 points at a Teleport proxy which users can access.
+```
+
+Copy the link shown after executing the above command and open the link in a web browser to complete user registration process (the link is `https://tele.teleporters.dev:443/web/invite/9b8c8ccb53d0aab612487d279d3d9831` in the above case).
+![Setup User](../../../../img/helm/digitalocean/setup-user.png)
+
+After you complete the registration process by setting up a password and enrolling in two-factor authentication, you will be logged in to Teleport WebUI. 
+
+In this step, we created a user **tadmin** with roles `access, edit`. These are the default roles available in Teleport. However, to allow this user to access the Kubernetes cluster, we will need to assign **tadmin** a role authorized to access the Kubernetes cluster. So first, let's create a role named **member** with the Kubernetes privilege `system:master`.
+
+**Create a new Kubernetes Role name "member":**
+
+Create a file name `member.yaml` with the following spec:
+```code
+$ vi member.yaml
+kind: role
+version: v4
+metadata:
+  name: member
+spec:
+  allow:
+    kubernetes_groups: ["system:masters"]
+
+```
+<Admonition
+  type="warning"
+  title="Warning"
+>
+`system:masters` is a Kubernetes built-in role that is similar to `sudo` permissions in Linux systems. It is used here for demonstration purposes and should be sparingly used in production.
+</Admonition>
+
+Next, create this role in Kubernetes with the command:
+
+```code
+
+$ POD=$(kubectl --namespace=teleport-cluster get pod -l app=teleport-cluster -o jsonpath='{.items[0].metadata.name}')
+$ kubectl --namespace=teleport-cluster exec -i ${POD?} -- tctl create -f < member.yaml
+```
+
+**Assign "member" role to user "tadmin":**
+
+Now we will assign Teleport user **tadmin** with this role. The example below shows a process using Teleport WebUI:
+First, lets select user edit menu:
+![Edit User](../../../../img/helm/digitalocean/edit-user.png)
+
+Second, update user role to assign **member** role:
+![Update Role](../../../../img/helm/digitalocean/update-role.png)
+
+We've update user **tadmin** with a role **member** whish is allowed to access Kubernetes cluster with privilege `system:master`.
+
+## Step 4/4. Connect to Kubernetes cluster using Teleport
+
+The following steps show how to access the Kubernetes cluster using `tsh`.
+
+**Login to Teleport:**
+First, login to Teleport using `tsh`.
+```code 
+$ tsh login --proxy=tele.teleporters.dev:443 --auth=local --user=tadmin(we created this user earlier)
```

Comment:
Rather than the `(we created this user earlier)` parenthetical, I think we should use a code comment